backend/src/utils/users.js

```diff
@@ -0,0 +1,49 @@
+const validateUsername = (username) => {
+    if (typeof username !== 'string' || username.length === 0) {
+        return 'Username is not valid';
+    }
+};
+
+const validateUserPhone = (phoneNumber) => {
+    if (typeof phoneNumber !== 'string' || phoneNumber.length === 0) {
```

Comment:
I think we should validate here that the phone number looks right, so its first character is + and everything after that is number so that it does not allow any spaces or anything.